core/kazoo_call/src/kapps_call_command.erl

```diff
@@ -1519,6 +1522,54 @@ b_play(Media, Terminators, Leg, Call) ->
 b_play(Media, Terminators, Leg, Endless, Call) ->
     wait_for_noop(Call, play(Media, Terminators, Leg, Endless, Call)).
 
+%%------------------------------------------------------------------------------
+%% @doc Produces the low level AMQP request to seek through the playing media.
+%% This request will execute immediately.
+%% @end
+%%------------------------------------------------------------------------------
+-spec seek(kapps_call:call()) -> kapps_api_std_return().
+seek(Call) ->
+    seek(?DEFAULT_SEEK_DURATION, Call).
+
+-spec seek(kz_term:api_integer(), kapps_call:call()) -> kapps_api_std_return().
+seek(Duration, Call) when Duration > 0 ->
+    seek(fastforward, Duration, Call);
+seek(Duration, Call) when Duration < 0 ->
+    seek(rewind, -Duration, Call);
+seek(_Duration, _Call) ->
+    ok.
+
+-spec b_seek(atom(), kz_term:api_pos_integer(), kapps_call:call()) -> kapps_api_std_return().
+b_seek(Direction, Duration, Call) ->
+    wait_for_noop(Call, seek(Direction, Duration, Call)).
+
+-spec seek(atom(), kz_term:api_pos_integer(), kapps_call:call()) -> kapps_api_std_return().
+seek(_Direction, 0, _Call) ->
+    ok;
+seek(Direction, Duration, Call) ->
+    NoopId = noop_id(),
+                                                %Commands = [kz_json:from_list([{<<"Application-Name">>, <<"noop">>}
+                                                %                               ,{<<"Call-ID">>, kapps_call:call_id(Call)}
+                                                %                               ,{<<"Msg-ID">>, NoopId}
+                                                %                              ])
+                                                %            ,seek_command(Direction, Duration)
+                                                %           ],
+                                                %Command = [{<<"Application-Name">>, <<"queue">>}
+                                                %           ,{<<"Commands">>, Commands}
+                                                %           ,{<<"Insert-At">>, <<"now">>}
+                                                %          ],
+    Command = seek_command(Direction, Duration),
+    send_command(Command, Call),
+    NoopId.
+
+-spec seek_command(atom(), kz_term:api_pos_integer()) -> kz_json:object().
+seek_command(Direction, Duration) ->
+    kz_json:from_list([{<<"Application-Name">>, <<"playseek">>}
+                      ,{<<"Direction">>,Direction}
```

Comment:
Space after the comma and before `Direction`. Same for `Duration` below

Reply:
@jamesaimonetti latest code has been updated for this, please take a look :)